gst/nnstreamer/tensor_query/tensor_query_client.c

```diff
@@ -690,8 +670,8 @@ gst_tensor_query_client_chain (GstPad * pad,
   g_free (val);
 
   if (NNS_EDGE_ERROR_NONE != nns_edge_send (self->edge_h, data_h)) {
-    nns_logw ("Failed to publish to server node, retry connection.");
-    goto retry;
+    nns_logi ("Failed to publish to server node, retry connection.");
+    goto done;
```

Comment:
Is it OK without changing ```res``` and returning it?

Reply:
It's Ok. Even if it fails to send the buffer, drop the current buffer and send the next incoming buffer.
And if we return error here, the pipeline stops.